viadot/flows/adls_to_azure_sql.py

```diff
@@ -82,6 +82,7 @@ def df_to_csv_task(df, remove_tab, path: str, sep: str = "\t"):
 def check_dtypes_sort(
     df: pd.DataFrame = None,
     dtypes: Dict[str, Any] = None,
+    apply: bool = True,
```

Comment:
This argument is applied only here. There is no way to modify it from the `Flow` perspective. Please add this argument to the flow `ADLSToAzureSQL`. 
Change the name at least in flow e.g.`check_dtypes_order` to have a clear name of what is for. `apply` may be hard to guess without documentation.

Reply:
I don't know what I was thinking about. Resolved!